Terminal.Gui/Drawing/Thickness.cs

```diff
@@ -108,9 +108,9 @@ public Thickness (int left, int top, int right, int bottom)
 		/// Gets whether the specified coordinates lie within the thickness (inside the bounding rectangle but outside of
 		/// the rectangle described by <see cref="GetInside(Rect)"/>.
 		/// </summary>
-		/// <param name="outside">Describes the location and size of the rectangle that contains the thickness.</param>
-		/// <param name="x"></param>
-		/// <param name="y"></param>
+		/// <param name="outside">The outside rect.</param>
```

Comment:
What was wrong with my description? I believe it more fully describes the parameter.  Just saying "The outside rect" is less complete.

Reply:
That was due to the merge conflict, sorry.